Neos.Flow/composer.json

```diff
@@ -41,6 +41,7 @@
         "doctrine/migrations": "^1.8",
         "doctrine/dbal": "^2.9",
         "doctrine/common": "^2.4",
+        "doctrine/persistence": "^1.3",
```

Comment:
Required due to `Doctrine\Persistence\Mapping\Driver\MappingDriver` in 40f35e0a3d0b12557621bf6498fb0b539c4d8494